configs/basecall.config

```diff
@@ -0,0 +1,38 @@
+/************************************************
+| CONFIGURATION FILE FOR NAO BASECALL WORKFLOW |
+************************************************/
+
+params {
+    mode = "basecall"
+    debug = true
+
+    // Directories
+    base_dir = "s3://nao-mgs-simon/NAO-ONT-20240905-DCS_RNA2_Test"
+
+    // Run parameters
+    // TODO: Automate getting run name from base_dir
+    nanopore_run = "NAO-ONT-20240905-DCS_RNA2_Test"
+
+    // TODO: Add optional paramter for ONT kit
+
+    // Files
+    pod5_dir = "${base_dir}/pod5_small/"
+    bam_dir = "${base_dir}/bam/"
+    calls_bam = "${base_dir}/bam/calls.bam"
+    fastq_file = "${base_dir}/raw/${nanopore_run}.fastq.gz"
+
+    // Default values for undefined run.nf parameters. Adding them here because they always get triggered
```

Comment:
I _think_ this should be fixed now that I've removed all the addParams statements. Try it again after merging in the changes from master?